src/dcap_provider.cpp

```diff
@@ -1422,6 +1423,22 @@ extern "C" quote3_error_t sgx_ql_get_quote_verification_collateral(
 
     try
     {
+        std::string collateral_version = get_collateral_version();
+        uint32_t version;
+        std::string api_version;
+
+        if (!collateral_version.compare("v1") ||
+            !collateral_version.compare("v2"))

```

Comment:
Nit: seeing these more and more everywhere, suggestion to make "v1" and "v2" etc. constants somewhere so there is more consistency and less room for error.

Reply:
Will fix